Formula/libhandy.rb

```diff
@@ -1,8 +1,8 @@
 class Libhandy < Formula
   desc "Building blocks for modern adaptive GNOME apps"
   homepage "https://gitlab.gnome.org/GNOME/libhandy"
-  url "https://gitlab.gnome.org/GNOME/libhandy/-/archive/1.7.0/libhandy-1.7.0.tar.gz"
-  sha256 "4a20ec0d3ba7c4743fee65d8421f2500bf627f73df5ee99699cdb4c54ccce3f0"
+  url "https://gitlab.gnome.org/GNOME/libhandy/-/archive/1.7.90/libhandy-1.7.90.tar.gz"
```

Comment:
This might a beta version.

Reply:
I was surprised too by the versioning and the version number that looked quite strange, but it does looks like it's not the case: https://gitlab.gnome.org/GNOME/libhandy/-/blob/main/NEWS

Reply:
Sounds like previous 1.7.0 was a development release and `libhandy` follows GNOME even/odd versioning https://gitlab.gnome.org/GNOME/libhandy/-/issues/465#note_1511780

Reply:
Looks like GNOME audit didn't work since we used Gitlab tarball rather than https://download.gnome.org/sources/libhandy/1.7/libhandy-1.7.0.tar.xz

Reply:
Switched in https://github.com/Homebrew/homebrew-core/commit/d9c33e8e3587d577187cabbf7e7f59a4adc79421. We should probably add https://gitlab.gnome.org to the audit.

Downgrading formulae is a pain. It might be better to just wait for 1.8.0.